src/hardware/vga_s3.cpp

```diff
@@ -479,11 +479,11 @@ Bitu SVGA_S3_ReadSEQ(Bitu reg,Bitu iolen) {
 	case 0x08:		/* PLL Unlock */
 		return vga.s3.pll.lock;
 	case 0x10:		/* Memory PLL Data Low */
-		return vga.s3.mclk.n || (vga.s3.mclk.r << 5);
+		return vga.s3.mclk.n | (vga.s3.mclk.r << 5);
 	case 0x11:		/* Memory PLL Data High */
 		return vga.s3.mclk.m;
 	case 0x12:		/* Video PLL Data Low */
-		return vga.s3.clk[3].n || (vga.s3.clk[3].r << 5);
+		return vga.s3.clk[3].n | (vga.s3.clk[3].r << 5);
```

Comment:
Can we flip the return fields so their order flows with the bits-fields? 0b111'00000 | 0b000'11111; (that's how I visually them being put back together into an 8-bit number).

`return (vga.s3.mclk.r << 5) | vga.s3.mclk.n;`

`return (vga.s3.clk[3].r << 5) | vga.s3.clk[3].n;`

(purely aesthetic :-)